plugin-test/copy-plugin/build/main.js

```diff
@@ -1 +1 @@
-(()=>{"use strict";var t={287:function(t,r,e){t.exports=e.p+"asset-modules/deepnested.txt"}},r={};function e(o){var i=r[o];if(void 0!==i)return i.exports;var n=r[o]={exports:{}};return t[o](n,n.exports,e),n.exports}e.g=function(){if("object"==typeof globalThis)return globalThis;try{return this||Function("return this")()}catch(t){if("object"==typeof window)return window}}(),(()=>{e.g.importScripts&&(t=e.g.location+"");var t,r=e.g.document;if(!t&&r&&(r.currentScript&&(t=r.currentScript.src),!t)){var o=r.getElementsByTagName("script");if(o.length){for(var i=o.length-1;i>-1&&(!t||!/^http(s?):/.test(t));)t=o[i--].src}}if(!t)throw Error("Automatic publicPath is not supported in this browser");t=t.replace(/#.*$/,"").replace(/\?.*$/,"").replace(/\/[^\/]+$/,"/"),e.p=t})();e(287)})();
\ No newline at end of file
+(()=>{"use strict";var t={287:function(t,r,e){t.exports=e.p+"asset-modules/deepnested.txt"}},r={};function e(o){var i=r[o];if(void 0!==i)return i.exports;var n=r[o]={exports:{}};return t[o](n,n.exports,e),n.exports}e.g=function(){if("object"==typeof globalThis)return globalThis;try{return this||Function("return this")()}catch(t){if("object"==typeof window)return window}}(),e.rv=function(){return"0.7.4"},(()=>{e.g.importScripts&&(t=e.g.location+"");var t,r=e.g.document;if(!t&&r&&(r.currentScript&&(t=r.currentScript.src),!t)){var o=r.getElementsByTagName("script");if(o.length){for(var i=o.length-1;i>-1&&(!t||!/^http(s?):/.test(t));)t=o[i--].src}}if(!t)throw Error("Automatic publicPath is not supported in this browser");t=t.replace(/#.*$/,"").replace(/\?.*$/,"").replace(/\/[^\/]+$/,"/"),e.p=t})(),(()=>{var t="function"==typeof e.rv?e.rv():"";e.ruid="bundler=rspack"+(t?"@"+t:"")})();e(287)})();
```

Comment:
`e.ruid="bundler=rspack"+(t?"@"+t:"")` -> `e.ruid="bundler=rspack@a.b.c")`

This makes it easier to get Rspack version by RegExp.